langchain/src/callbacks/base.ts

```diff
@@ -510,6 +510,12 @@ export class CallbackManager extends BaseCallbackManager {
     return manager;
   }
 
+  copy(): CallbackManager {
```

Comment:
Might as well accept here as an argument another list of handlers to add?

Reply:
hmm yeah good point